<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wallpapers &amp; More Asset Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_workingWith">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Device&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="Main_treeView">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="Image_graphicsView">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="Image_Info_TextBrowser"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="UploadToAppButton">
            <property name="text">
             <string>Upload To Application</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Remove_Button">
            <property name="text">
             <string>Remove Assest</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="UploadToDatabaseButton">
      <property name="text">
       <string>Upload New Assets</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Assets"/>
    <addaction name="actionCheck_For_Duplicates_2"/>
    <addaction name="separator"/>
    <addaction name="actionClose_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_WM"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionDevice"/>
    <addaction name="actionDatabase"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionAbout_WM">
   <property name="text">
    <string>About WM</string>
   </property>
  </action>
  <action name="actionCheck_For_Duplicates">
   <property name="text">
    <string>Check For Duplicates</string>
   </property>
  </action>
  <action name="actionNew_Assets">
   <property name="text">
    <string>New Assets</string>
   </property>
  </action>
  <action name="actionCheck_For_Duplicates_2">
   <property name="text">
    <string>Check For Duplicates</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>